w3c.json.html

```diff
@@ -64,44 +64,44 @@ <h1>The <code>w3c.json</code> file</h1>
       For integration purposes, please use your <em>GitHub</em> ID.
     </dd>
 
-    <dt id="repo-type"><code>repo-type</code></dt>
+    <dt id="repo-type" data-type="string"><code>repo-type</code></dt>
     <dd>
     String to identify the type and purpose of the repository.
     The possible values for this field are:<br/><br/>
     <dl>
-      <dt>rec-track</dt>
+      <dt class='value'>rec-track</dt>
```

Comment:
What does this class `value` do?

Also:
```suggestion
      <dt class="value">rec-track</dt>
```
ie, double quotes instead, for consistency? (Same in all other instances below.)

Reply:
@plehegar :arrow_up:

Reply:
@plehegar, there are still many instances of `class='value'` (you changed just one), and I don't understand your comment (*&ldquo;should be a v.next request&rdquo;*)&hellip;

Reply:
sorry. got mixed up in the comments. The call value is only there to identify specific values in the doc. Nothing harmful as far as I can tell.